build.gradle

```diff
@@ -18,8 +18,8 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "com.google.protobuf:protobuf-gradle-plugin:$protobufVersion"
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$navVersion"
-        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.2.1"
-        classpath "io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.9.1"
+        classpath "org.jlleitschuh.gradle:ktlint-gradle:9.4.1"
```

Comment:
Why does "Allowing trailing commas", require 127 files to be changed?

A lot of the reformatting is unnecessary or worse imho, and the potential for merge conflicts is huge.

Reply:
I appreciate the effort :heart:  but would like to reduce this PR to "just allowing trailing commas".

Reply:
Well I looked for a way to enable only the training commas did not find a rule that was preventing it in our config file.
After some Googling I found out that it is only possible by upgrading the plugin

Reply:
I don't want to decide this. We should check with the rest of the team.

Reply:
Fair enough

Reply:
i also fear a lot of merge conflicts. is this a reasonable price for trailing commas?